src/index.css

```diff
@@ -395,4 +392,22 @@ code {
 
 .category-sidebar .menu-list a.is-active {
     background-color: #70bdf4;
-}
\ No newline at end of file
+}
+
+.mcq-answers {
+    margin-left: 1em !important;
```

Comment:
avoid the use of important.

Reply:
the rule is overwritten by `.content ul { margin-left: 2em; }` from bulma.
I removed my rule for now, as the question component will require some styling anyway.